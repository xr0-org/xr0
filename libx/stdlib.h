#ifndef STDLIB_H
#define STDLIB_H

axiom void *
malloc(int size) [ .alloc result; ];

axiom void
free(void *ptr) [ .dealloc ptr; ];

#endif
