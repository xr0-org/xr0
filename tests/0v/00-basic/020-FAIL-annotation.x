#include <stdlib.h>

void *
leak()
{
	void *p;

	p = malloc(1);
	return p;
}
