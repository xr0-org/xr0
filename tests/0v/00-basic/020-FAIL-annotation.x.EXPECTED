leak | #1 âˆˆ {?0}: actual and abstract states differ
